grapher/axis/Axis.ts

```diff
@@ -26,7 +26,10 @@ interface Tickmark {
 abstract class AbstractAxis {
     config: AxisConfig
     @observable.ref domain: ValueRange
-    @observable formatColumn?: CoreColumn // Pass the column purely for formatting reasons. Might be a better way to do this.
+    @observable tickFormatter?: (
```

Comment:
I actually prefer passing the column, the more I think about it. But seems like we should pass it as `column` instead of `formatColumn` (which jus sounds weird).

The really nice thing about having all formating done in one place (the column level), is hat i s really easy o test.